matsim/src/main/java/org/matsim/vehicles/VehicleUtils.java

```diff
@@ -75,7 +71,12 @@ public static Vehicles createVehiclesContainer() {
 	}
 
 	public static VehicleType getDefaultVehicleType() {
```

Comment:
Ì would suggest to rename this method to `createDefaultVehicleType()`, because the user should not expect to receive _the_ single static `VehicleType` object anymore.

Reply:
I agree, changed!